CITATION.cff

```diff
@@ -0,0 +1,19 @@
+cff-version: 1.2.0
+message: "If you are referring to the STAC specification in your publications, please cite it as below."
+type: software
+title: "SpatioTemporal Asset Catalog (STAC) specification"
+authors:
+  - name: "STAC Project Steering Committee"
```

Comment:
Should this be SpatioTemporal Asset Catalog PSC instead? Is the steering committee more of a governing body than an author? Is there a difference?

Reply:
I think the way it is makes sense and gives the most reasonable "citation line" with not so much "repetition".

Reply:
GDAL says authors are contributors. https://github.com/OSGeo/gdal/blob/master/CITATION

Maybe something like 
```type: software
title: "SpatioTemporal Asset Catalog (STAC) specification"
authors:
  - name: "STAC Contributors"
  ```

Reply:
Yes, that would also work. :-) I have no preference.

Reply:
+1 on STAC Contributors, suggested that as changes.